src/style.css

```diff
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-weight: 900;
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-weight: 300;
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+}
+
+.large__bar {
+  display: flex;
+  width: 100%;
+  margin: 20px 0;
+}
+
+.large__icon {
+  display: block;
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 45px;
+  left: 34px;
+}
+
+.input__large {
+  display: block;
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: inherit;
+}
+
+.input__large:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input__large:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
```

Comment:
But you have the same border for the default state. Do you need to change it on focus?